README.md

```diff
@@ -17,11 +17,11 @@ Rasgo Transforms provide jinja SQL templates that can be applied to your data us
 # Running Transforms
 
 Rasgo Transforms can be applied via:
-- your Rasgo Feature Store account ([pyrasgo](https://pypi.org/project/pyrasgo/) - Account required) 
+- your Rasgo Feature Store account ([pyrasgo](https://pypi.org/project/pyrasgo/) - Account required)
```

Comment:
@cnash6 We're moving away from calling Rasgo a Feature store right? Shouldn't we change the text here then?